Paralayout.podspec

```diff
@@ -8,8 +8,4 @@ Pod::Spec.new do |s|
   s.source   = { :git => 'https://github.com/square/Paralayout.git', :tag => s.version }
   s.source_files = 'Paralayout/*.{swift}'
   s.ios.deployment_target = '13.0'
-
-  s.test_spec 'Tests' do |test_spec|
-    test_spec.source_files = 'ParalayoutTests/*{.swift}'
-  end
```

Comment:
I ended up removing this after all in 6602c13 so that we wouldn't have to support Cocoapod's construction of unit test targets. IMO we don't gain anything from having this here given that we have other CI runs executing the test suite.